src/components/GraphFilter/GraphFilter.tsx

```diff
@@ -65,7 +134,7 @@ export default class GraphFilter extends React.PureComponent<GraphFilterPropsRea
         <Toolbar>
           <FormGroup className={zeroPaddingLeft}>
             {this.props.node ? (
-              <Button className={namespaceStyle} onClick={this.props.onNamespaceReturn}>
+              <Button className={namespaceStyle} onClick={() => this.handleNamespaceReturn()}>
```

Comment:
nitpick, probably cleaner to do: `onClick={this.handleNamespaceReturn}`.

Reply:
+1, I will fix in future.